sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -2351,6 +2355,22 @@ public ReadChangeStream withBacklogReplicationAdjustment(Duration adjustment) {
       return toBuilder().setBacklogReplicationAdjustment(adjustment).build();
     }
 
+    /**
+     * Returns a new {@link BigtableIO.ReadChangeStream} that overrides timeout for ReadChangeStream
+     * requests.
+     *
+     * <p>This is useful to override the default of 15s timeout if the checkpoint duration is longer
+     * than 15s. Setting this value to longer than periodic checkpoint duration ensures that
```

Comment:
Maybe we should note that you probably want to add some padding beyond checkpoint duration to avoid deadline exceeded when checkpointing is slow

Reply:
Done